packages/editor/src/ui/BottomToolbar/NodeTools.tsx

```diff
@@ -10,22 +10,22 @@ export type BottomToolbarMainBarProps = {
 export const BottomToolbarMainBar: React.FC<BottomToolbarMainBarProps> = React.memo(
   ({ nodeId, actionsLeft }) => {
     const { title, icon } = usePluginOfCell(nodeId) ?? {};
-
+    const { t } = useUiTranslator();
     return (
       <div>
         <Grid container={true} direction="row" alignItems="center">
           {icon || title ? (
             <Grid item={true}>
               <Avatar
-                children={icon || (title ? title[0] : '')}
+                children={icon || t(title ? title[0] : '')}
```

Comment:
the point here is that the avatar shows the first letter of the title if no icon is defined

so with translation it would be

`icon || t(title)[0]`

Reply:
I think to remove `t` wrapper here may be better?